templates/Places/complaints.html

```diff
@@ -89,14 +95,20 @@
         }
 
     </style>
+      <!-- Bootstrap  -->
+     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
Remove these. Are these scripts required ?

Reply:
Yes sir this is not needed I checked without this.